docs/guides/jupyter.rst

```diff
@@ -0,0 +1,363 @@
+How to use Jupyter notebooks in Sphinx
+======================================
+
+`Jupyter`_ notebooks are a popular tool to describe computational narratives
+that mix code, prose, images, interactive components, and more.
+Embedding them in your Sphinx project allows using these rich documents as documentation,
+which can provide a great experience for tutorials, examples, and other types of technical content.
+There are a few extensions that allow integrating Jupyter and Sphinx,
+and this document will explain how to achieve some of the most commonly requested features.
+
+.. _Jupyter: https://jupyter.org/
+
+By default, `Sphinx only supports reStructuredText source files for
+documentation <https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-source_suffix>`_.
+Luckily, there are a few extensions to also allow Jupyter notebooks as source files.
+
+.. _nb-background:
+
+Background on the existing relevant extensions
+----------------------------------------------
+
+There are two main extensions that add support Jupyter notebooks as source files in Sphinx:
+nbsphinx_ and MyST-NB_. They have similar intent and basic functionality:
+both can read notebooks in ``.ipynb`` and additional formats supported by `jupytext`_,
+and are configured in a similar way.
+However, there are some differences between them:
+
+- nsphinx uses `pandoc <https://pandoc.org/>`_ to convert the Markdown from Jupyter notebooks
+  to reStructuredText and then to docutils AST,
+  whereas MyST-NB uses `MyST-Parser`_ to directly convert the Markdown text to docutils AST.
+  Both Markdown flavors are mostly equal, but they have some differences.
```

Comment:
Rather than focusing on the "parser", I'd probably focus on the "flavor" of markdown that each supports. Most users don't really know what a "parser" is anyway, and rather they care more about "what syntax can I write?". I'd re-focus around "CommonMark/Jupyter markdown" with nbsphinx and MyST Markdown with MyST-NB.

Reply:
I think talking about "which parser" might be interesting technical information difficult to find elsewhere, but you're right that I should more explicitly differentiate what this actually means in practical terms. I will rework this part.